docs/fhir/fhir_import_config.rst

```diff
@@ -53,8 +53,8 @@ FHIRResourceType instances (rendered as "FHIR Resource Types" in Django
 Admin) to configure mapping; FHIRImportConfig uses
 FHIRImportResourceType instances ("FHIR Import Resource Types").
 
-To see what this looks like, navigate to FHIR > FHIR Importer Resource
-Types, and click "Add FHIR Importer Resource Type".
```

Comment:
I actually do kinda wish this was called the "FHIR Importer".  I had trouble understanding what `FHIRImportResourceType` means - it's a resource type for a FHIR Importer, not a command to import a resource type